client/src/utils_links.js

```diff
@@ -24,7 +24,22 @@ const ENTITY_TYPE_TO_RELATED_OBJECT_TYPE = flip(
 )
 
 export function getEntityInfoFromUrl(url) {
-  const urnMatch = url.match(/^urn:anet:([^:]*):(.*)$/)
+  const urlObject = new URL(url, window.location.href)
+  if (urlObject.protocol === "urn:") {
+    return getEntityFromUrlPattern(
+      url,
+      urlObject.pathname,
+      /^anet:([^:]*):(.*)$/
+    )
+  }
+  if (urlObject.hostname === window.location.hostname) {
```

Comment:
Probably more correct to do:
```
  if (urlObject.host === window.location.host) {
```
since that includes both the hostname and the port number (different sites may be running on the same host but with different port numbers).

Reply:
Applied